RFC: Delegated  Identity API implementation (#2481)
spiffe/spire: doc/spire_agent.md

    }
}
```
## Delegated Identity API

> SPIRE agent has support for Delegated Identity API. This API is intended for use cases where a (authorized) workload wants access to the X509-SVIDs and bundles on behalf of another workload. A list of authorized delegates SPIFFE IDs of workloads are defined for this purpose. The API is served over the same Unix domain socket as the admin API socket. Based on workload's selectors, a (authorized) workload could be subscribed to get X509-SVIDs and federation trust domains from the registration entries, which matches to the provided selectors.

Reviewer: I think that it would be more accurate to refer to federated _bundles_, not trust domains. I have a suggestion here
```suggestion
SPIRE agent has support for Delegated Identity API. This API is intended for use cases where a (authorized) workload wants access to the X509-SVIDs and bundles on behalf of another workload. A list of authorized delegates SPIFFE IDs of workloads are defined for this purpose. The API is served over the same Unix domain socket as the admin API socket. Based on workload's selectors, a (authorized) workload could be subscribed to get X509-SVIDs and federated bundles from the registered entries that match the provided selectors.
```
